build-system/babel-plugins/babel-plugin-transform-preact-props-json-object/index.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = function (babel) {
+  const {types: t} = babel;
+
+  return {
+    name: 'ast-transform', // not required
+    visitor: {
```

Comment:
Can we restrict these to Preact code only? I can see "props" being used by accident in non-React contexts.